### PR TITLE
fix: add allowBreakglass in both go and java client

### PR DESCRIPTION
**Pull request by sqin2019**
part of #203 
tested the env var locally.

### CODE REVIEW

File: clients/go/apis/v1alpha1/config.go [hunk 1]
```diff
@@ -367,6 +367,10 @@ type Justification struct {
 	// all manual calls are expected to pass in a justification in the "justification-token" header.
 	// If omitted, justifications will not be added to logs, even if provided.
 	Enabled bool `yaml:"enabled,omitempty" env:"JUSTIFICATION_ENABLED,overwrite"`
+
+	// AllowBreakglass indicates whether JVS should allow unsigned justification.
+	// If justification is not enabled, AllowBreakglass will be ignored, even if provided.
```
Comment (sethvargo) [inline clients/go/apis/v1alpha1/config.go:372]: Please document the default behavior.

File: clients/go/apis/v1alpha1/config.go [hunk 2]
```diff
@@ -367,6 +367,10 @@ type Justification struct {
 	// all manual calls are expected to pass in a justification in the "justification-token" header.
 	// If omitted, justifications will not be added to logs, even if provided.
 	Enabled bool `yaml:"enabled,omitempty" env:"JUSTIFICATION_ENABLED,overwrite"`
+
+	// AllowBreakglass indicates whether JVS should allow unsigned justification. If omitted, it is set to false.
```
Comment (yolocs) [inline clients/go/apis/v1alpha1/config.go:371]:
```suggestion
	// AllowBreakglass indicates whether JVS should allow unsigned justification. By default, it's false.
```
nit

**Review by yolocs — COMMENTED** [review]
Don't we need to modify https://github.com/abcxyz/lumberjack/blob/main/clients/go/pkg/justification/processor.go to allow/disallow breakglass?

**Comment by sqin2019** [comment]
> Don't we need to modify https://github.com/abcxyz/lumberjack/blob/main/clients/go/pkg/justification/processor.go to allow/disallow breakglass?

this logic is in JVS client, https://github.com/abcxyz/jvs/blob/74efb454b360cfc45e6564f5c780883ed3c4209a/client-lib/java/src/main/java/com/abcxyz/jvs/JvsClient.java#L86

**Comment by yolocs** [comment]
I see!

**Review by yolocs — APPROVED** [review]

**Pull request merged**